_topic_maps/_topic_map.yml

```
@@ -742,8 +742,8 @@ Topics:
     File: dynamic-plug-in-example
   - Name: Dynamic plugin reference
     File: dynamic-plug-ins-reference
-- Name: Web terminal
-  File: odc-about-web-terminal
+- Name: Using the web terminal
+  File: odc-web-terminal
```

```suggestion
  File: odc-using-web-terminal
```
- Should the file be more reflective of the name?

---

Good point. No it's not too invasive to change, thanks 🙂

---

OH, I remember why I didn't change this initially - I actually meant to, but the one I've renamed to "accessing the web console" used to be called "using the web console", I guess I forgot that I wanted to rename this after changing that. Thank you!